eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/Parser/Common.php

```diff
@@ -168,6 +168,7 @@ public function mapConfig(array &$scopeSettings, $currentScope, ContextualizerIn
             $contextualizer->setContextualParameter('binary_dir', $currentScope, $scopeSettings['binary_dir']);
         }
         if (isset($scopeSettings['api_keys']['google_maps'])) {
+            $contextualizer->setContextualParameter('api_keys', $currentScope, ['google_maps' => $scopeSettings['api_keys']['google_maps']]);
             $contextualizer->setContextualParameter('api_keys.google_maps', $currentScope, $scopeSettings['api_keys']['google_maps']);
```

Comment:
@alongosz you miss read me, what the added line does is make sure any code asking for whole `api_keys` hash also works, like done in Platform UI. I assume you refer to anything requesting `api_keys.google_maps` specifically, which already worked from before.